src/components/screens/wallet/index.js

```diff
@@ -100,6 +104,7 @@ const Wallet = ({ t, history }) => {
           tabName={t('Transactions')}
           tabId="Transactions"
           t={t}
+          isWallet={history.location.pathname === '/wallet'}
```

Comment:
Prefer to use `routes.wallet.path` from `constants/routes.js`

Reply:
Done